conda-forge.yml

```diff
@@ -20,4 +15,5 @@ github_actions:
   - pull_request
 provider:
   linux_64: github_actions
```

Comment:
i think because we use cross compilation, this is the line that matters.

you can change this to azure. but i "triggered" a job for you.

However, if you aren't getting through the full build, azure is fine (6hr mark right...)